### PR TITLE
Fix: Import

### PR DESCRIPTION
**Pull request by localheinz**
This pull request

- [x] imports a class

### CODE REVIEW

File: tests/unit/Util/TestClassTest.php [hunk 1]
```diff
@@ -1024,7 +1025,7 @@ public function testWithDuplicateKeyDataProviders(): void
     public function testTestWithEmptyAnnotation(): void
     {
         $result = DocBlock::ofMethod(new ReflectionMethod(
-            \VariousDocblockDefinedDataProvider::class,
+            VariousDocblockDefinedDataProvider::class,
             'anotherAnnotation'
         ), VariousDocblockDefinedDataProvider::class)->getProvidedData();
```
Comment (localheinz) [inline tests/unit/Util/TestClassTest.php:1030]: Not sure why this class has not been imported before.

**Comment by sebastianbergmann** [comment]
Merged, thanks. However, I cannot merged this from `8.5` to `9.5` without conflicts. Can you send a PR for `9.5`? Thanks!

**Comment by localheinz** [comment]
Thank you, @sebastianbergmann - follow up is coming in a moment!

**Comment by localheinz** [comment]
@sebastianbergmann 

Fix is not necessary for `9.5`, already applied in one way or another!

**Pull request merged**